src/render_cmd.rs

```diff
@@ -287,6 +293,11 @@ impl RenderCmd {
         }
         Ok(buffer)
     }
+
+    /// Check if deployment name is filtered
+    fn is_name_filtered(&self, regex: String, name: &str) -> bool {
```

Comment:
`regex` should also be accepted as `&str`; like you did with `name`. Also, to avoid the call to `unwrap()`, return a `Result` type:

```rust
/// Check if deployment name is filtered
fn is_name_filtered(&self, regex: &str, name: &str) -> anyhow::Result<bool> {
    Ok(Regex::new(regex)?.is_match(name))
}
```

These are the steps I took to refactor:
1. turn the `regex` param into `regex: &str`
2. `Regex::new(&regex)` -> `Regex::new(regex)`
3. Use `?` to return a possible error `Regex::new(regex)?`
4. Change the return type from `bool` to `anyhow::Result<bool>`
5. Wrap the command in `Ok`